analysis/GreenSKU-Framework/figures/paper_figures_original/Table_V.csv

```diff
@@ -0,0 +1,6 @@
+Config,Operational Savings,Embodied Savings,Carbon Savings
```

Comment:
Instead of saving these things, I think you can just have them output in the notebook.